dashboard/test/integration/session_cookie_test.rb

```diff
@@ -12,12 +12,6 @@ class SessionCookieTest < ActionDispatch::IntegrationTest
     assert cookies['_learn_session_test']
   end
 
-  test 'no cookies if you do not do anything' do
-    get '/'
-
-    assert_nil cookies['_learn_session_test']
-  end
-
```

Comment:
This was removed because the rails session is now instantiated even on the home page to track stable id. This was true even before this change because if you hit `/` it would redirect you to `/sign_in` and that will vend you a cookie.